### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jiji14**

### CODE REVIEW

**Comment by jiji14** [comment]
***

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
...
+animal_list = ["hare", "rabbit", "tortoise", "goat", "hound", "duck", "ducking", "fox", "lion", "boar", "ant", "grasshopper", 
+                "cat", "monkey", "dog", "mouse", "crow", "eagle", "wolf", "peacock"]
+
+def isAnimal(value):
+    if value in animal_list:
+        value = True    
+    else:
+        value = False
+        
+isAnimal("hare")
```
Comment (atn832) [inline README.md:14]: There is a bug in isAnimal. Have you tried print(isAnimal("hare"))? In the console it prints out "None". But I expect it to print out "True".

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
...
+
+# count words
+
+def countWords(text):
+    word_list = text.split()
+    dic = {}
+    for word in word_list:
+        dic[word] = dic.get(word, 0) + 1
+    
+countWords("i like carrots like rabbits")
```
Comment (atn832) [inline README.md:27]: Same issue: print(countWords("....")) prints out None. You forgot to return a value.

Comment (atn832) [reply README.md:27]: Also, we usually write one pull request per task. This helps work on several things at the same time. For example if countWords works but isAnimal does not yet, you can merge the first pull request while you keep working on the other.

**Review by atn832 — COMMENTED** [review]
Finally the code should be written in a .py file.

**Comment by atn832** [comment]
Closing this pull request since you've submitted the same code in other pull requests.

**Pull request closed without merging**